xbmc/platform/darwin/ios/Info.plist.in

```diff
@@ -119,7 +119,7 @@
 	<true/>
 	<key>UIRequiredDeviceCapabilities</key>
 	<array>
-		<string>armv7</string>
+		<string>armv64</string>
```

Comment:
Are you sure?

Reply:
No I’m not

Reply:
I am now: https://developer.apple.com/documentation/bundleresources/information_property_list/uirequireddevicecapabilities

Reply:
Whoops, extra `v`